source/index.html.md.erb

```diff
@@ -23,4 +23,6 @@ You can [read further documentation about how GOV.UK One Login works](/how-gov-u
 
 Contact us if you have any questions on our [#govuk-one-login Slack channel](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC).
 
+<%= partial "partials/changelog" %>
```

Comment:
would it be a partial or embedded into the index page?

Reply:
I left it as a partial so that its all the same sort of stuff in one place and hopefully easier to maintain